pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -29,8 +29,17 @@ mod tests;
 
 #[frame_support::pallet]
 pub mod pallet {
-	use frame_support::pallet_prelude::*;
-	use frame_system::{ensure_root, ensure_signed, pallet_prelude::OriginFor};
+	use frame_support::{
+		pallet_prelude::{DispatchResult, *},
+		traits::Currency,
+	};
+	use frame_system::{
+		ensure_root, ensure_signed, ensure_signed_or_root, pallet_prelude::OriginFor,
+	};
+
+	type BalanceOf<T> =
+		<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
+	type BlockNumberOf<T> = <T as frame_system::Config>::BlockNumber;
```

Comment:
Let's remove this alias and use `T::BlockNumber` from `frame_system::Config` directly
```suggestion
```